frontend/src/metabase/visualizations/components/ClickActions/ClickActionsPopover.styled.tsx

```diff
@@ -12,6 +12,8 @@ export const Container = styled.div`
 
 export const FlexTippyPopover = styled(TippyPopover)`
   display: flex;
+  max-height: 80vh;
```

Comment:
This makes the entire popover scrollable, which is not ideal and is inconsistent with other popovers.

If you try filtering on a Vendor column, you'll notice that the popover has a fixed max height (unrelated to viewport height) and only the content part is scrollable (the footer and header are always visible) ([image](https://github.com/user-attachments/assets/22b5f6a0-e002-4b1d-a306-b619a51bcaaa)).
I believe we should fix it in the same way here.

Reply:
It's consistent with the popover that shows up when you click an existing filter tag (ie. `Title contains 267 selections` in [this question](http://localhost:3000/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjoxLCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjgsImZpbHRlciI6WyJjb250YWlucyIseyJjYXNlLXNlbnNpdGl2ZSI6ZmFsc2V9LFsiZmllbGQiLDY1LHsiYmFzZS10eXBlIjoidHlwZS9UZXh0In1dLCJKYW1lcyIsIk9saXZpYSIsIkV0aGFuIiwiU29waGlhIiwiTGlhbSIsIkVtbWEiLCJOb2FoIiwiQXZhIiwiV2lsbGlhbSIsIklzYWJlbGxhIiwiTWFzb24iLCJNaWEiLCJCZW5qYW1pbiIsIkFtZWxpYSIsIkx1Y2FzIiwiSGFycGVyIiwiRWxpamFoIiwiQ2hhcmxvdHRlIiwiQWxleGFuZGVyIiwiQWJpZ2FpbCIsIkhlbnJ5IiwiRW1pbHkiLCJNaWNoYWVsIiwiRXZlbHluIiwiSmFjb2IiLCJMaWx5IiwiRGFuaWVsIiwiQXJpYSIsIlNlYmFzdGlhbiIsIkVsbGEiLCJNYXR0aGV3IiwiQ2hsb2UiLCJEYXZpZCIsIkdyYWNlIiwiU2FtdWVsIiwiWm9lIiwiSm9zZXBoIiwiSGFubmFoIiwiSmFja3NvbiIsIlBlbmVsb3BlIiwiSmFjayIsIkxheWxhIiwiQWlkZW4iLCJTY2FybGV0dCIsIk93ZW4iLCJFbGxpZSIsIld5YXR0IiwiU3RlbGxhIiwiR2FicmllbCIsIk5vcmEiLCJDYWxlYiIsIk1pbGEiLCJJc2FhYyIsIkF1cm9yYSIsIkxvZ2FuIiwiQWRkaXNvbiIsIk5hdGhhbiIsIkJlbGxhIiwiRHlsYW4iLCJMdWN5IiwiQ2hyaXN0aWFuIiwiVmljdG9yaWEiLCJMdWtlIiwiSGF6ZWwiLCJMZXZpIiwiU2F2YW5uYWgiLCJBZGFtIiwiTGlsbGlhbiIsIkFudGhvbnkiLCJQYWlzbGV5IiwiSnVsaWFuIiwiQXVkcmV5IiwiQ2FydGVyIiwiVmlvbGV0IiwiQ2hhcmxlcyIsIk5hdGFsaWUiLCJFenJhIiwiU2FtYW50aGEiLCJUaG9tYXMiLCJMZWFoIiwiR3JheXNvbiIsIkJyb29rbHluIiwiQW5kcmV3IiwiQ2Fyb2xpbmUiLCJDb25ub3IiLCJSaWxleSIsIkRvbWluaWMiLCJNYWRlbGluZSIsIlNvcGhpZSIsIkV2YW4iLCJBYWxpeWFoIiwiQ29sdG9uIiwiSnVsaWEiLCJKb3JkYW4iLCJTZXJlbml0eSIsIkh1bnRlciIsIlNhZGllIiwiRWxpIiwiS2VubmVkeSIsIk5pY2hvbGFzIiwiR2FicmllbGxhIiwiQ2hhc2UiLCJDbGFpcmUiLCJBYXJvbiIsIk1heWEiLCJSeWFuIiwiQWxleGlzIiwiSm9uYXRoYW4iLCJBZHJpYW4iLCJDaHJpc3RvcGhlciIsIlBleXRvbiIsIkpheG9uIiwiUnVieSIsIkplcmVtaWFoIiwiTWlsZXMiLCJJc2xhIiwiTGluY29sbiIsIkFsaWNlIiwiUm9iZXJ0IiwiSHVkc29uIiwiSmFkZSIsIkFzaGVyIiwiTGVvIiwiRWxlbmEiLCJJc2FpYWgiLCJOYXRoYW5pZWwiLCJFdmEiLCJDYW1lcm9uIiwiU2FyYWgiLCJFdmVyZXR0IiwiQ29vcGVyIiwiV2VzdG9uIiwiQmVubmV0dCIsIkNhcnNvbiIsIlF1aW5uIiwiR2lvdmFubmkiLCJBdGhlbmEiLCJBdXN0aW4iLCJMeWRpYSIsIkF4ZWwiLCJHYWJyaWVsbGUiLCJQYXJrZXIiLCJEYWlzeSIsIkl2eSIsIkRlY2xhbiIsIk1hcmlhIiwiUm9tYW4iLCJJc2FiZWwiLCJLYXlkZW4iLCJFbWlsaWEiLCJTYXd5ZXIiLCJXaWxsb3ciLCJCZWF1IiwiRWxpYW5hIiwiSmFzb24iLCJOYW9taSIsIlphbmUiLCJKb3NlcGhpbmUiLCJNaWNhaCIsIlJlYmVjY2EiLCJCcm9va3MiLCJBZGVsaW5lIiwiR2F2aW4iLCJDZWNpbGlhIiwiU3BlbmNlciIsIlZhbGVudGluYSIsIkhhcnJpc29uIiwiVml2aWFuIiwiVmluY2VudCIsIkJyYXh0b24iLCJNYWRkb3giLCJBbGxpc29uIiwiVHJhdmlzIiwiQXJpYW5hIiwiQW5nZWxpbmEiLCJKYWtlIiwiTWF4d2VsbCIsIktpbnNsZXkiLCJQYXRyaWNrIiwiQ2xhcmEiLCJGaW5uIiwiRWxlYW5vciIsIkt5bGUiLCJDb3JhIiwiSnVzdGluIiwiTWFyZ290IiwiSXZhbiIsIkRlbGlsYWgiLCJCcnljZSIsIkVzdGhlciIsIkNhbWRlbiIsIkJyaWFubmEiLCJNYXgiLCJBdmVyeSIsIk1hbGFjaGkiLCJSb3NlIiwiUnlkZXIiLCJGYWl0aCIsIkRpZWdvIiwiU2llbm5hIiwiQ29saW4iLCJFbHNpZSIsIkthbGViIiwiVHJlbnQiLCJFYXN0b24iLCJMdW5hIiwiQmxha2UiLCJNaWxsaWUiLCJMdWlzIiwiSXNhYmVsbGUiLCJKb3JkeW4iLCJKZXJlbXkiLCJNYWRkaWUiLCJaYW5kZXIiLCJGaW9uYSIsIkVsaWFzIiwiTWFkZWx5biIsIkphY2UiLCJPbGl2ZSIsIkRldmluIiwiTHVjaWEiLCJUeSIsIkxlaWxhbmkiLCJNYWJlbCIsIkFydGh1ciIsIkhvcGUiLCJNYXJrIiwiQXVicmV5IiwiVHJldm9yIiwiTWFyY3VzIiwiU2V0aCIsIlBhaWdlIiwiRGFtaWFuIiwiU2lsYXMiLCJFbWVyeSIsIkVtbWV0dCIsIklyaXMiLCJDb2xlIiwiWm9leSIsIlN0ZXZlbiIsIkVsaXNlIiwiUnViZW4iLCJKb2VsIiwiVGVzc2EiLCJBbGV4IiwiRXNtZSIsIkJhaWxleSIsIkVsbGlvdCIsIkF1dHVtbiIsIktpZXJhbiIsIkNhbGxpZSIsIkphcmVkIiwiRmVsaWNpdHkiLCJSZWVzZSIsIlNhYnJpbmEiLCJHcmFkeSIsIlNlcmVuYSIsIkFyYWJlbGxhIiwiVHJpc3RhbiIsIkt5bGllIiwiRGVsYW5leSIsIkdlbW1hIiwiUmVpZCIsIlNreWUiLCJUcmV5IiwiUG9wcHkiLCJCcmVudCJdfX0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9LCJ0eXBlIjoicXVlc3Rpb24ifQ==)).


<img width="1474" alt="Screenshot 2024-10-28 at 22 02 15" src="https://github.com/user-attachments/assets/3f538055-668b-4b03-a44e-727d57ed5790">


I agree that having the footer always be visible would be better but I wanted to get a fix out for this bug first (and this fix is not inconsistent with existing UI).

@mngr @AlessioLaiso Maybe we can have a (small) design sprint on how this could look? We could make the entire popover scrollable (like Kamil proposed), or we could make the input scrollable.

Reply:
There are pros and cons in both approaches. I agree that the main improvement would be to have the footer be always visible. We can dig deeper if we believe this is a common scenario, but if we think that having so many selections in a filter is an edge case, I'd keep it simple and make sure it's at least functional - and this PR achieves that.

Reply:
Works great! :+1: